compose/container.py

```diff
@@ -172,6 +172,19 @@ def has_api_logs(self):
         log_type = self.log_driver
         return not log_type or log_type in ('json-file', 'journald')
 
+    @property
+    def create_health_status_string(self):
+        """ Generate UP status string with up time and health
+        """
+        status_string = 'Up'
+        if 'Health' in self.get('State'):
+            container_status = self.get('State.Health.Status')
```

Comment:
Because `get` will return `None` if the key is undefined, you can simplify this part of the code by writing
```python
container_status = self.get('State.Health.Status')
if container_status == 'starting':
  ...
elif container_status is not None:
  ...
```